### DISCUSSION TITLE
Menu is blue and not readable !

### DISCUSSION THREAD

**Discussion by HinataNekoNya** [thatskymod/Sky-CotL-Scripts#160] [Q&A]
Look !
![image](https://github.com/user-attachments/assets/c1b882ca-e442-491c-8518-b6b6190be787)
![image](https://github.com/user-attachments/assets/bd12ecde-2390-4778-be0b-2cced7d2f66c)

**Comment by Slingexe** [comment]
Turn off your HDR Mode


https://support.microsoft.com/en-us/windows/hdr-settings-in-windows-2d767185-38ec-7fdc-6f97-bbc6c5ef24e6

**Reply by HinataNekoNya** [reply]
Why HDR do this why I don't Correct the application?
